project/BinaryIncompatibilities.scala

```diff
@@ -8,6 +8,15 @@ object BinaryIncompatibilities {
   )
 
   val Linker = Seq(
+      // private[emitter], not an issue
```

Comment:
Exclude all of `backend`? (maybe we should even have these excludes constantly, it seems unnecessary to maintain these lists).